src/main/resources/assets/twilightforest/lang/en_ud.lang

```diff
@@ -0,0 +1,622 @@
+twilightforest.config.portal_creator=ɯǝʇI uoᴉʇɐǝɹƆ ꞁɐʇɹoԀ
+twilightforest.config.portal_creator.tooltip=˙ꞁɐʇɹoԀ ʇsǝɹoℲ ʇɥᵷᴉꞁᴉʍ⟘ ǝɥʇ ǝʇɐǝɹɔ oʇ pǝsn ɯǝʇᴉ ɟo ᗡI ᵷuᴉɹʇS ʎɹʇsᴉᵷǝᴚ
+twilightforest.config.portal_creator_meta=ɐʇǝW s,ɯǝʇI uoᴉʇɐǝɹƆ ꞁɐʇɹoԀ
+twilightforest.config.portal_creator_meta.tooltip=˙ɐʇɐpɐʇǝɯ ʎuɐ ɹoɟ ⥝- ˙ꞁɐʇɹoԀ ʇsǝɹoℲ ʇɥᵷᴉꞁᴉʍ⟘ ǝɥʇ ǝʇɐǝɹɔ oʇ pǝsn ɯǝʇᴉ ɟo ɐʇǝW
+twilightforest.config.portal_lighting=ǝɹᴉℲ ᵷuᴉuʇɥᵷᴉꞀ uoᴉʇɐǝɹƆ ꞁɐʇɹoԀ
+twilightforest.config.portal_lighting.tooltip=˙unɟ ǝʞᴉꞁ ʇ,uop oɥʍ ǝsoɥʇ ɹoℲ ˙ǝɹᴉɟ uo sᵷuᴉɥʇ ʇǝs ʇou oʇ ꞁɐʇɹod ǝɥʇ sdɐz ʇɐɥʇ ᵷuᴉuʇɥᵷᴉꞁ ǝɥʇ ʇuɐʍ noʎ ɟᴉ ǝnɹʇ sᴉɥʇ ʇǝS
+
+twilightforest.config.uncrafting=ǝꞁqɐ⟘ ᵷuᴉʇɟɐɹɔu∩ uᴉ ᵷuᴉʇɟɐɹɔu∩ ǝꞁqɐsᴉᗡ
+twilightforest.config.uncrafting.tooltip=˙sǝdᴉɔǝɹ ǝꞁqɐʇᴉoꞁdxǝ sǝɔnpoɹd spoɯ ɹǝɥʇo ɥʇᴉʍ uoᴉʇɔɐɹǝʇuᴉ uǝɥʍ uoᴉʇdo uɐ sɐ pǝpᴉʌoɹԀ ˙ǝꞁqɐʇ ᵷuᴉʇɟɐɹɔun ǝɥʇ ɟo uoᴉʇɔunɟ ᵷuᴉʇɟɐɹɔun ǝɥʇ ǝꞁqɐsᴉᗡ
+
+twilightforest.config.antibuilder_blacklist=ʇsᴉꞁʞɔɐꞁᗺ ɹǝpꞁᴉnqᴉʇuⱯ
+twilightforest.config.antibuilder_blacklist.tooltip=˙ꞁɐuoᴉʇdo sᴉ ɐʇǝɯ (ɐʇǝɯ:ʞɔoꞁq:uᴉɐɯop) ˙ʇsᴉꞁʞɔɐꞁq ɹǝpꞁᴉnᗺ-ᴉʇuⱯ
+
+twilightforest.config.animate_trophyitem=ꞁǝpoɯ ɯǝʇI uo spɐǝH ʎɥdoɹ⟘ ǝʇɐʇoᴚ
+twilightforest.config.animate_trophyitem.tooltip=˙unɟ ǝʞᴉꞁ ʇ,uop oɥʍ ǝsoɥʇ ɹoℲ ˙ꞁꞁɐ ʇɐ ʇɔɐdɯᴉ ǝɔuɐɯɹoɟɹǝd ou sɐH ˙ꞁǝpoɯ ɯǝʇᴉ uo spɐǝɥ ʎɥdoɹʇ ǝʇɐʇoᴚ
+
+twilightforest.config.loading_screen=uǝǝɹɔS ᵷuᴉpɐoꞀ
+twilightforest.config.loading_screen.tooltip=uǝǝɹɔs ᵷuᴉpɐoꞀ ǝɥʇ ɹoɟ sꞁoɹʇuoƆ :ʎꞁuo ʇuǝᴉꞁƆ
+twilightforest.config.loading_icon_enable=uoᴉʇɐɯᴉuⱯ uoɔI ᵷuᴉpɐoꞀ ǝꞁqɐuƎ
+twilightforest.config.loading_icon_enable.tooltip=˙unɟ ǝʞᴉꞁ ʇ,uop oɥʍ ǝsoɥʇ ɹoℲ ˙ꞁꞁɐ ʇɐ ʇɔɐdɯᴉ ǝɔuɐɯɹoɟɹǝd ou sɐH ˙uoɔᴉ ᵷuᴉpɐoꞀ ǝɥʇ ǝꞁqqoM
+twilightforest.config.loading_screen_swap_frequency=ᵷuᴉɥɔʇᴉʍs uǝǝɹɔS ᵷuᴉpɐoꞀ
+twilightforest.config.loading_screen_swap_frequency.tooltip=˙ꞁꞁɐ ʇɐ ǝꞁɔʎɔ ʇou oʇ 0 oʇ ʇǝS ˙ǝᵷuɐɥɔ uǝǝɹɔs ᵷuᴉpɐoꞁ ɥɔɐǝ uǝǝʍʇǝq sʞɔᴉʇ ʎuɐɯ ʍoH
+twilightforest.config.loading_icon_wobble_bounce_frequency=pǝǝdS uoᴉʇɐɯᴉuⱯ uoɔI
+twilightforest.config.loading_icon_wobble_bounce_frequency.tooltip=˙ǝɔunoq puɐ ǝꞁqqoʍ ɟo ʎɔuǝnbǝɹℲ
+twilightforest.config.loading_icon_scale=ᵷuᴉꞁɐɔS uoɔI
+twilightforest.config.loading_icon_scale.tooltip=˙uoɔᴉ ᵷuᴉpɐoꞁ ʎɔunoq ǝꞁoɥʍ ɟo ǝꞁɐɔS
+twilightforest.config.loading_icon_bounciness=ssǝuᴉɔunoᗺ uoɔI
+twilightforest.config.loading_icon_bounciness.tooltip=˙sǝɔunoq uoɔᴉ ᵷuᴉpɐoꞁ ǝɥʇ ɥɔnɯ ʍoH
+twilightforest.config.loading_icon_tilting=ssǝuᴉꞁqqoM uoɔI
+twilightforest.config.loading_icon_tilting.tooltip=˙sǝꞁqqoʍ uoɔᴉ ᵷuᴉpɐoꞁ ǝɥʇ ɹɐɟ ʍoH
+twilightforest.config.loading_icon_tilt_pushback=ʇǝsɟɟO ʇꞁᴉ⟘ uoɔI
+twilightforest.config.loading_icon_tilt_pushback.tooltip=˙uoɔᴉ ᵷuᴉpɐoꞁ ɟo ǝꞁqqoʍ ǝɥʇ ɹǝʇuǝɔ-ǝɹ oʇ ǝnꞁɐʌ ʞɔɐqɥsnԀ
+
+twilightforest.book.lichtower.1=ǝsɹnɔ ǝɥʇ ɯoɹɟ ɔᴉᵷɐɯ ǝɥ⟘ ˙ǝɹoɟǝq uǝǝs ǝʌ,I ʎuɐ uɐɥʇ ɹǝᵷuoɹʇs 'ǝsɹnɔ ɐ ʎq pǝʇɔǝʇoɹd ǝɹɐ ɹǝʍoʇ ǝɥʇ ɟo sʞɔᴉɹq ǝɥ⟘ ˙ɹǝʍoʇ sᴉɥʇ ᵷuᴉpunoɹɹns ɐɹnɐ ǝᵷuɐɹʇs ǝɥʇ ᵷuᴉuᴉɯɐxǝ unᵷǝq ǝʌɐɥ I\n\n0ⱢⱯ00\u[[sɹǝʇsuoɯ ʎq uo pǝʍɐuᵷ 'ʞooqǝʇou s,ɹǝɹoꞁdxǝ uⱯ]]8ⱢⱯ00\u
```

Comment:
pretty sure all of these are broken. Stuff like `\u00A78` got flipped as well as the order of some of the formatting codes